WebPhapp/backend/server.js

```diff
@@ -39,6 +39,9 @@ function convertDatesToString(prescription){
     prescription.writtenDate = new Date(prescription.writtenDate).toString();
     prescription.fillDates = prescription.fillDates.filter(dateInt => dateInt > 0);
     prescription.fillDates = prescription.fillDates.map(dateInt => new Date(dateInt).toString());
+    if(prescription.cancelDate > 0){
```

Comment:
I would appreciate a comment here about the cancelDate defaulting to `-1`. Took me some time to understand why this works.